server/authorizer/authorizer_test.go

```diff
@@ -41,6 +41,9 @@ func TestBasic(t *testing.T) {
         allow = allow_inner {
             not input.path[0] = "undefined" # testing undefined
             not conflict_error              # testing eval errors
+            not input.path[0] = "not_allowed_with_reason"
+			not	input.path[0] = "wrong_result_set"
```

Comment:
[nit] Something with tabs and spaces is odd here -- that's always icky with inline rego code, I'm afraid. I'm sure it looks OK in your editor, but could you please adjust it so it is consistent?